forum/forum/settings.py

```diff
@@ -150,11 +152,33 @@
 
 REST_FRAMEWORK = {
     'DEFAULT_RENDERER_CLASSES': [
-        'rest_framework.renderers.JSONRenderer', 
+        'rest_framework.renderers.JSONRenderer',
+        'rest_framework_simplejwt.authentication.JWTAuthentication',
     ],
 }
 
 SECURE_BROWSER_XSS_FILTER = True
 SECURE_CONTENT_TYPE_NOSNIFF = True
 SECURE_HSTS_SECONDS = 3600  
 SECURE_SSL_REDIRECT = os.getenv('DJANGO_SECURE_SSL_REDIRECT', 'True') == 'True'
+
+
+# JWT settings
+
+SIMPLE_JWT = {
+    'ACCESS_TOKEN_LIFETIME': timedelta(minutes=60),
+    'REFRESH_TOKEN_LIFETIME': timedelta(days=30),
+    'ROTATE_REFRESH_TOKENS': True,
+    'BLACKLIST_AFTER_ROTATION': False,
```

Comment:
Consider setting `BLACKLIST_AFTER_ROTATION` to `True` if you want to invalidate old refresh tokens after they are used.
If `BLACKLIST_AFTER_ROTATION` is `True`, ensure `rest_framework_simplejwt.token_blacklist` should be added to `INSTALLED_APPS`:
```
INSTALLED_APPS += [
    'rest_framework_simplejwt.token_blacklist',
]
```